nixops/backends/libvirtd.py

```diff
@@ -217,10 +224,15 @@ def read_file(stream, nbytes, f):
             stream.sendAll(read_file, f)
             stream.finish()
 
+    def _qemu_executable(self):
+        domaincaps_xml = self.conn.getDomainCapabilities(
+            emulatorbin=None, arch='x86_64', machine=None, virttype='kvm',
```

Comment:
hardcoding arch could be a problem, couldn't it ?

Reply:
The current version has hardcoded `qemu-system-x86_64`, so I decided to let it be for now it and maybe address it later in a separate PR. Didn't want to include too many unrelated changes in a single PR.